srv/components/controller/files/scripts/provision.sh

```diff
@@ -1090,21 +1169,44 @@ fw_upgrade_health_check()
 fw_update()
 {
     _error=0
-    
+
     fw_upgrade_health_check
     pfu_enable
-    ftp_enable
+    #ftp_enable
+    sftp_enable #TBD
```

Comment:
sftp_enable => sftp_enabled

Reply:
this is function, to enable the sftp service not the sftp_enabled flag.